portable/MemMang/heap_5.c

```diff
@@ -442,7 +442,7 @@ static void prvInsertBlockIntoFreeList( BlockLink_t * pxBlockToInsert )
 void vPortDefineHeapRegions( const HeapRegion_t * const pxHeapRegions )
 {
     BlockLink_t * pxFirstFreeBlockInRegion = NULL, * pxPreviousFreeBlock;
-    size_t xAlignedHeap;
+    portPOINTER_SIZE_TYPE xAlignedHeap;
     size_t xTotalRegionSize, xTotalHeapSize = 0;
     BaseType_t xDefinedRegions = 0;
     size_t xAddress;

```

Comment:
Should this be portPOINTER_SIZE_TYPE as well?

Reply:
If it must be  compatible with C89  then it's true.  Another way - use uintptr_t, but it since  C99 only

Reply:
I fixed it